pkg/kubelet/kubelet.go

```diff
@@ -1938,6 +1949,7 @@ func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 func (kl *Kubelet) HandlePodUpdates(pods []*v1.Pod) {
 	start := kl.clock.Now()
 	for _, pod := range pods {
+		kl.secretManager.RegisterPod(pod)
```

Comment:
I'm missing some context for when Reconcile is called. But since it only call UpdatePod(), I guess it's not adding new pods... @yujuhong - WDYT?